openfisca_core/entities/group_entity.py

```diff
@@ -23,7 +27,15 @@ class GroupEntity(Entity):
 
     """  # noqa RST301
 
-    def __init__(self, key, plural, label, doc, roles, containing_entities=()):
+    def __init__(
+        self,
+        key: str,
+        plural: str,
+        label: str,
+        doc: str,
+        roles: Sequence[HasKey],
+        containing_entities: Sequence[str] = (),
+    ) -> object:
```

Comment:
Since this is a constructor, couldn't we be more specific? 🙂 
```suggestion
    ) -> GroupEntity:
```

Reply:
This was a typo that was already there, thanks!